### PR TITLE
Style slider for new content editor HTML

### PR DESCRIPTION
**Pull request by jessouyangbraven**
**Task**: https://app.asana.com/0/1170776727341290/1174547828723905

**Test**

- Create empty sections, add slider question, rate this module, and a slider in a table
- Here's the code tab input I tested with: https://pastebin.com/DJfd1hPp
- Save and publish to local canvasweb

**Slider question**
- Before CSS change the slider in a question doesn't take up the full width:
Screen Shot 2020-05-14 at 3 41 40 PM

- After CSS change, pasted into Chrome console, slider at full width:
Screen Shot 2020-05-14 at 3 42 05 PM

**Rate this module**: looks the same as before
Screen Shot 2020-05-14 at 3 37 40 PM

**Slider input in table**: looks the same as before
Screen Shot 2020-05-14 at 3 54 02 PM

**Details**

- The CSS is looking for `<div class="slider-container range">` to apply the slider styling, which we're not generating from the new editor. Instead, we look for a range input within a question and apply the full-width CSS.

### CODE REVIEW

File: braven_newui.css [hunk 1]
```diff
@@ -2642,6 +2642,11 @@ table.sort-to-match tr {
 	width: 100%;
 }
 
+.module-block > div.question input[type=range] {
```
Comment (jessouyangbraven) [inline braven_newui.css:2645]: Can I delete  `.slider-container.range input[type=range] {` above? I tested and didn't see any differences, but I wasn't sure if there was some existing module HTML I didn't know about.

**Comment by rshipp** [comment]
I'm not sure how much it matters, but I think the sliders in the old modules all use this `two-thirds` class, so they don't take up the whole width:

Screen Shot 2020-05-15 at 9 30 52 AM

Is this CSS that you're using applied anywhere? I have a feeling it might just be old stuff that's no longer used.

**Comment by jessouyangbraven** [comment]
I also just noticed that the display value and the slider are flipped. It's probably a `dont-mix` issue with elements in `<fieldset>`? Will put up a separate PR for that. 

> I'm not sure how much it matters, but I think the sliders in the old modules all use this two-thirds class, so they don't take up the whole width:

I totally missed the two-thirds class you mentioned in the task. Derp. 

So, I went ahead and added the class on the CKE side, hoping to avoid a CSS change. But we run into the same class-collision problem that makes the slider widget un-selectable and un-editable. :\

I can update the styling in this PR to make it match `two-thirds` and I think it'll be okay?

> Is this CSS that you're using applied anywhere? I have a feeling it might just be old stuff that's no longer used.

Those classes are still being inserted by Rate This Module and sliders inserted in content blocks.

**Comment by rshipp** [comment]
Ah interesting, ok. 

> It's probably a dont-mix issue with elements in <fieldset>?

Great catch, I hadn't even thought of that.

File: braven_newui.css (hunk 1 above)
Comment (rshipp) [reply braven_newui.css:2645]: I assume it's for rate-this-module, so if that still works after this two-thirds change, its probably fine? There's a lot of css in here we no longer use and was just never cleaned up.

**Pull request merged**